app/models/retirement_manager.rb

```diff
@@ -0,0 +1,14 @@
+class RetirementManager
+  def self.check
+    ems_ids = MiqServer.my_server.zone.ext_management_system_ids
+    [LoadBalancer, OrchestrationStack, Vm, Service].flat_map do |i|
+      instances = not_retired_with_ems(i, ems_ids)
+      instances.each(&:retirement_check)
+    end
+  end
+
+  private_class_method def self.not_retired_with_ems(model, ems_ids)
```

Comment:
Prefer we use the syntax of placing `private_class_method` on the line below the method with the method name passed as a symbol to be consistent with the rest of the project.

Example: https://github.com/ManageIQ/manageiq/blob/470292c508d8da6cbc90d7be6b72f3df09ea0653/app/models/metric/capture.rb#L234